app/views/groups/index.html.erb

```diff
@@ -0,0 +1,20 @@
+<div class='d-block mx-auto'>
+  <% if @groups.empty? %>
+  <h5> Not group for this user </h5>
+  <% else %>
+    
+    <% @groups.each do |group| %>
+   <div class="row p-3">
+    <div class="col-md-3"> </div>
+      <a href='<%= group_expends_path(group) %>' class='p-3 col-md-6 text-secondary d-flex justify-content-between border border-1'>
+        <%= image_tag("#{group.icon}.png", :alt => "picture") %>
+        <h3><%= group.name %> </h3>
+        <h6><%= group.expends.where(author_id: current_user.id).sum(:amount) %> $ </h6>
+      </a>
+      <div class="col-md-3"> </div>
+      </div>
+    <% end %>
+  <% end %>
+
+   </div>
+   <a href='<%= new_group_path %>' class='fixed-bottom btn btn-primary green mt-3 mb-3 p-3 d-block mx-auto'> Create group </a>
```

Comment:
- [ ] Please make sure this button looks like that in the original design. It should have a white border surrounding the text.
### Your Implementation
![image](https://github.com/francksefu/my_expended/assets/45460192/60b2b897-0b7e-4f91-beda-a22400c4da27)
### Required
![image](https://github.com/francksefu/my_expended/assets/45460192/49ccbb67-d101-426b-82df-1752b7cc3057)